docs/source/tools/assistant.rst

```diff
@@ -133,7 +133,7 @@ You can control the build by changing your project's ``daml.yaml``:
 ``source``
   The path to the source code.
 
-The generated ``.dar`` file is created in ``dist/${name}.dar`` by default. To override the default location, pass the ``-o`` argument to ``daml build``::
+The generated ``.dar`` file is created in ``.daml/dist/${name}.dar`` by default. To override the default location, pass the ``-o`` argument to ``daml build``::
```

Comment:
Does this need reflecting in any quickstart guides we have?

Reply:
No the quickstart guides still use `target/daml/iou.dar` since the Java codegen `pom.xml` hardcodes that path and nobody has bothered changing it so far.

Reply:
Do we have a ticket for this o(l|d)d b behaviour of the codegen? If not, we should probably write one.

Reply:
I’ll just fix it instead of making a ticket :slightly_smiling_face: